packages/dev/s2-docs/pages/s2/styling.mdx

```diff
@@ -253,6 +199,11 @@ const styles = style({
 Extract common styles into constants and spread them into `style` calls. These must be in the same file or imported from another file as a macro.
 
 ```tsx
+// style-utils.ts
+export const bannerBackground = () => 'blue-1000' as const;
+
+// component.tsx
+import {bannerBackground} from './style-utils' with {type: 'macro'};
```

Comment:
Kinda confusing IMO. You also showed custom utilities in a separate code block below. Do we need this one too?